pkg/client/monitoring/v1/types.go

```diff
@@ -629,6 +637,10 @@ type AlertmanagerSpec struct {
 	// object, which shall be mounted into the Alertmanager Pods.
 	// The Secrets are mounted into /etc/alertmanager/secrets/<secret-name>.
 	Secrets []string `json:"secrets,omitempty"`
+	// ConfigMaps is a list of ConfigMaps in the same namespace as the Alertmanager
+	// object, which shall be mounted into the Alertmanager Pods.
+	// The ConfigMaps are mounted into /etc/alertmanager/configmaps/<configmap-name>.
```

Comment:
Missing some details mentioned in the `PrometheusSpec.ConfigMaps` comment:

```
ConfigMap changes after initial creation of a Prometheus object are not reflected in the running Pods. To change the ConfigMaps mounted into the Prometheus Pods, the object must be deleted and recreated with the new list of ConfigMaps.
```

Reply:
I actually tested this, and this statement became false when we introduced the reconciliation on illegal updates. Will remove where applicable.